insert-ordered-containers.cabal

```diff
@@ -46,6 +46,7 @@ library
     , base-compat           >=0.10.5  && <0.12
     , hashable              >=1.2.6.1 && <1.4
     , lens                  >=4.17    && <4.19
+    , optics                >=0.2     && <0.3
```

Comment:
`optics-extra` would be enough, wouldn't it?

Reply:
optics-core + optics-extra. Should I change it?

Reply:
Yes please. Slightly less dependencies: good.

(In fact I thought of splitting out the `lens` (and `optics`) stuff, but that could wait).